src/amqp_string.pyx

```diff
@@ -45,19 +44,18 @@ cdef class AMQPString(StructBase):
 
     def __str__(self):
         as_bytes = c_strings.STRING_c_str(self._c_value)
-        if six.PY3:
-            try:
-                return as_bytes.decode('UTF-8')
-            except UnicodeDecodeError:
-                pass
+        try:
+            return as_bytes.decode('UTF-8')
+        except UnicodeDecodeError:
+            pass
         return str(as_bytes)
```

Comment:
I've kept this return in here in case there is an error with unicode decode. Wasn't sure if it can be removed